tests/test_bootstrap.py

```diff
@@ -109,7 +109,7 @@ async def test_bootstrap_license(
                         "secretKeyRef": {"key": "license", "name": f"license-{name}"},
                     },
                     "name": "my-crate-cluster",
-                    "version": "4.1.5",
+                    "version": "4.3.3",
```

Comment:
note: `4.3.4` would be the latest `4.3.x` release, while `4.4.0` is the latest stable

Reply:
Good point, I will change it to 4.4.0 in a subsequent PR if that's ok.

Reply:
Haha true. I just approved a few minutes before `4.4.0` was out. 😂